<launch>
    <node name="galaxy_camera" pkg="galaxy_camera" type="galaxy_camera" output="screen">
        <param name="image_width" value="1280"/>
        <param name="image_height" value="1024"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value=""/>
        <param name="camera_frame_id" value="camera"/>

        <!-- Exposure -->
        <param name="exposure_auto" value="false"/>
        <param name="exposure_value" value="12500"/>
        <param name="exposure_min" value="20"/>
        <param name="exposure_max" value="1000000"/>

        <!-- Gain -->
        <param name="gain_auto" value="true"/>
        <param name="gain_value" value="13"/>
        <param name="gain_min" value="0"/>
        <param name="gain_max" value="16"/>

        <!-- Frame rate -->
        <param name="frame_rate" value="15"/>

        <!-- White balance -->
        <param name="white_auto" value="true"/>
        <!-- 0, 1, 2 for red, green, blue-->
        <param name="white_selector" value="0"/>
        <param name="white_value" value="1"/>

        <!-- Gamma Value -->
        <param name="gamma_value" value="0"/>

        <!-- image improve 0-all 1- -->
        <param name="improve_mode" value="0"/> 
    </node>
</launch>
